component/presentation.go

```diff
@@ -54,9 +54,7 @@ func (p *RepoPresentation) Render() *vecty.HTML {
 					vecty.Property(atom.Height.String(), "36"),
 				),
 			),
-			elem.Div(
-				p.presentationChangesAndError(),
-			),
+			p.presentationChangesAndError(),
```

Comment:
Is this change neccessary? Why? What would happen if it weren't done?

Reply:
Prior to the change [`vecty.List`](https://godoc.org/github.com/gopherjs/vecty#List) included the ability to represent markup, so `presentationChangesAndError` returning `vecty.Markup(vecty.Style("word-break", "break-word")),` as an element in the `vecty.List` was valid.

However, since it is now defined as `[]vecty.ComponentOrHTML` (no longer includes markup), that is no longer valid. You could change the return type of `presentationChangesAndError` from `vecty.List` to `[]vecty.MarkupOrChild` to achieve the same code layout you had here before, but I felt that was probably needlessly ugly compared to just returning `*HTML`.

Reply:
Thanks a lot for explaining that, it's very helpful for me to understand the change.

After thinking about this, I still prefer to keep the method `presentationChangesAndError` returning just the content, and not the wrapper div. The wrapper div should be here, wrapping the output of the function. I'll make that change myself.

Another bonus of doing that is that it'll make the API change PR diff smaller.